src/platform/packages/shared/kbn-esql-ast/src/commands_registry/registry.ts

```diff
@@ -59,9 +64,10 @@ export interface ICommandMethods<TContext = any> {
    */
   columnsAfter?: (
     command: ESQLCommand,
-    previousColumns: ESQLFieldWithMetadata[],
-    context?: TContext
-  ) => ESQLFieldWithMetadata[];
+    previousColumns: ESQLColumnData[],
+    query: string,
```

Comment:
Why do we need the query in columnsAfter?

Reply:
To get the column name when there is an implicit declaration e.g. `EVAL FLOOR(1)`